src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = (props) => {
+  const { movies } = props;
```

Comment:
```suggestion
export const MoviesList: React.FC<Props> = ({ movies }) => {
```
 u can connect these lines

Reply:
+